SCClassLibrary/Common/Streams/Rest.sc

```diff
@@ -1,56 +1,23 @@
-Rest {
-	var <>dur = 1;
-	*new { |dur(1)|
-		^super.newCopyArgs(dur)
-	}
-	// for use by Pfunc and Pfuncn
-	*processRest { |inval|
-		inval.put(\isRest, true);
-		^1
-	}
-	processRest { |inval|
-		inval.put(\isRest, true);
-		^dur.value(inval)
-	}
-	// for use e.g. in ListPatterns
-	*embedInStream { |inval|
-		^this.processRest(inval).yield;  // the actual return value is the next inval
-	}
-	embedInStream { |inval|
-		^this.processRest(inval).yield;
-	}
-	*asStream {
-		^Routine({ |inval|
-			loop {
-				inval = this.embedInStream(inval);
-			}
-		})
-	}
-	asStream {
-		^Routine({ |inval|
-			loop {
-				inval = this.embedInStream(inval);
-			}
-		})
+
+Rest : Operand  {
+
+	*new { |value = 1|
+		^super.newCopyArgs(value.dereferenceOperand).unwrapBoolean
```

Comment:
functionally, yes, in this instance, they're the same. but from a software engineering point of view, this constructor relies on knowledge of the order of declaration of its superclass's instance variables. and from the non-SC-specialist's point of view, that behavior is not documented. in fact, it is documented incorrectly:

> If the superclass's new method requires arguments, the first arguments passed to newCopyArgs will be passed on to that method, and the following arguments will be copied to this class's instance variables.

and its usage here lacks semanticity, since `new` is a more concise and frequent statement which would have the same effect. i already gave this PR an approved review below; i'm not "holding it up." i'm pointing out a questionable practice; you are welcome to leave this code in its current state.